### ISSUE TITLE
[BUG] TiledMap Parser error while set a tmx file

### ISSUE THREAD

**Issue by doums** [cocos/cocos-engine#1829]
Hi,

I try to make a TiledMap whit cocos creator. When I drag my tmx file from asset panel to TiledMap component of the TiledMap node in the properties panel I get this error : 

```
Error: Please load the resource first : C:/Users/Pierre/Documents/CocosProjects/FeudalTimes/assets/map/tile_set/iso-outside.tsx
at parseXMLFile (C:\CocosCreator\resources\engine\cocos2d\tilemap\CCTMXXMLParser.js:566:27)
at parseXMLFile (C:\CocosCreator\resources\engine\cocos2d\tilemap\CCTMXXMLParser.js:646:22)
at parseXMLString (C:\CocosCreator\resources\engine\cocos2d\tilemap\CCTMXXMLParser.js:943:21)
at initWithXML (C:\CocosCreator\resources\engine\cocos2d\tilemap\CCTMXXMLParser.js:554:21)
at ctor (C:\CocosCreator\resources\engine\cocos2d\tilemap\CCTMXXMLParser.js:273:18)
at new TheClass (C:\CocosCreator\resources\engine\cocos2d\core\platform\_CCClass.js:117:34)
at initWithXML (C:\CocosCreator\resources\engine\cocos2d\tilemap\CCSGTMXTiledMap.js:274:23)
at cc_TiledMap._applyFile (C:\CocosCreator\resources\engine\cocos2d\tilemap\CCTiledMap.js:851:30)
at cc_TiledMap.set (C:\CocosCreator\resources\engine\cocos2d\tilemap\CCTiledMap.js:251:26)
at cc_TiledMap.r.set [as tmxAsset] (C:\CocosCreator\resources\app.asar\editor\page\scene-utils\asset-watcher.js:1:697)
```

However the .tsx files are in the rigth place under C:/Users/Pierre/Documents/CocosProjects/FeudalTimes/assets/map/tile_set/iso-outside.tsx

What is wrong ?

edit: I used Tiled for make the iso map with this settings:

- isometric
- width 50
- height 50
- tile width 64
- tile height 32
- tile Side Lenght (Hex) 0
- Stagger Axis Y
- Stagger Inder odd
- tile layer format base64 (zlib)
- tile renderer order bottom right
- background color no set

I use 3 layers and one object layer
All layers have at least one tile. The object layer have several object
I use one tileSet based on Tileset image

here my map if you want to test it https://www.mediafire.com/?x33zqhk1hjmddyj

Cocos Creator v1.5.2 - windows10

EDIT: I tried to load the same map in a fresh cocos2d-x c++ project, last release, in both Android Studio and Linux, no problem. The map is correctly loaded and works fine.

**Comment by knoxHuang** [comment]
Sorry, TSX format is not supported for the time being.

**Comment by doums** [comment]
Hum okay, but is not possible to generate .tmx without .tsx with Tiled Editor. And tsx contain texture informations. So it is not possible to have a functionnal tiledMap without their .tsx

So when you said 

> Sorry, TSX format is not supported for the time being.

it is like you said "tiledMap is not supported for the time being."
It is the whole tileMap component which is not supported.

It's a really big probem !

**Comment by knoxHuang** [comment]
You're right.
Sorry, this is caused by the bug of the engine loader. （Will be repaired as soon as possible）

**Comment by doums** [comment]
Ok thank you for info.

**Comment by jareguo** [comment]
the problem will be fixed in 1.6, thanks

**Comment by doums** [comment]
> Hum okay, but is not possible to generate .tmx without .tsx with Tiled Editor.

After more work on Tiled I realized it is false. We can create tmx with tsx informations embedded in tmx. Sorry for that mistake.
